desktop/components/commercial_filter/url_handler.coffee

```diff
@@ -1,18 +1,21 @@
 Backbone = require 'backbone'
 qs = require 'qs'
+collectPageTitle = require './page_title.coffee'
+{ PAGE_TITLE_FILTERS } = require('sharify').data
 
 module.exports = class UrlHandler extends Backbone.Router
 
   initialize: ({ @params }) ->
     throw new Error 'Requires a params model' unless @params?
 
-    @listenTo @params, 'change', @setURL
+    @listenTo @params, 'change', @setURLAndTitle
```

Comment:
i guess in the event that further filters are selected the title can get out of sync... so i see the UX need for this now. :+1:

Reply:
Yea, exactly.

Also, even on the initial load, we trigger a 'change' to get everything to render, which causes this to revert almost immediately- which is odd. A _possible_ fix would be to refactor that initial trigger (which is more of a convenience to trigger renders), or to avoid the first trigger causing a change in this router, etc...

But yea, it was easy enough to just support it client-side as well!

Reply:
OK makes sense. I think it's fine to support it client-side too in that case.